ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -429,7 +436,13 @@ object BigDiffy extends Command {
     @tailrec
     def get(xs: Array[String], i: Int, r: GenericRecord): String =
       if (i == xs.length - 1) {
-        String.valueOf(r.get(xs(i)))
+        val valueOfKey = r.get((xs(i)))
+        if (valueOfKey == null) {
+          logger.warn(
+            s"""Null value found for key: ${xs.mkString(".")}.
+               | If this is not expected check your or use a different key.""".stripMargin)
```

Comment:
"check your key" instead of "check your"?

Reply:
oops thanks should be "check your data"